internal/serverless/client.go

```diff
@@ -42,12 +42,10 @@ var (
 
 func NewClient(opts ...ClientOption) (*Client, error) {
 	hostEnvName := ServerlessHostvironmentVariable
-	host, ok := os.LookupEnv(hostEnvName)
-	if !ok {
-		host = defaultHostURL
-	}
+	host := os.Getenv(hostEnvName)
 	if host == "" {
-		return nil, fmt.Errorf("unable to obtain value from %s environment variable", hostEnvName)
+		logger.Debugf("Using default host URL: %s", defaultHostURL)
```

Comment:
Nit. Maybe we can print in any case what endpoint is used.

Reply:
Moved this logger.Debug after creating the client with all the options.

I also ensured that environment variables (EC_HOST) has preference over the configuration set in the profile (e.g. `~/.elastic-package/profiles/serverless/config.yml`)